ingestion/src/metadata/ingestion/source/dashboard/metabase/metadata.py

```diff
@@ -60,6 +62,10 @@ def __init__(
     ):
         super().__init__(config, metadata_config)
         self.metabase_session = self.client["metabase_session"]
+        self.metabaseClient = MetabaseClient(
```

Comment:
we should be already getting the metabase client in `self.client` no need to reinitialise it

Reply:
Hi @ulixius9 , this part is not clear to me, could you provide additional context here? I created the metabaseClient class to move REST calls to the client and now you are suggesting to not have the metabase client initialized here.

So could you suggest how to approach the problem of moving REST calls away from metadata.py without initializing Metabase client ?

Reply:
Hi @Hemal-Mamtora, the `MetabaseSource` extends the `DashboardServiceSource` in within the init of `DashboardServiceSource` we initialise the variable [self.client](https://github.com/open-metadata/OpenMetadata/blob/905201548df1fce4ed33b76ca25aa64e9215c3db/ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py#L177)  which basically stores the result of [get_connection](https://github.com/open-metadata/OpenMetadata/blob/905201548df1fce4ed33b76ca25aa64e9215c3db/ingestion/src/metadata/ingestion/source/dashboard/metabase/connection.py#L33) 

so when you will address the above comment to update the get_connection method to return the object of `MetabaseClient` you will not need to initialise it once again here you can just use the variable `self.client` instead of `self.metabaseClient `